### PR TITLE
[Oy2-8982] Create AMB-CH Measure

### PR DESCRIPTION
**Pull request by jphechter**
## Purpose

Create the AMB-CH measure using commonly used question components.

#### Linked Issues to Close

[Oy2-8982](https://qmacbis.atlassian.net/browse/OY2-8982?atlOrigin=eyJpIjoiNTY4YzhlN2U4NGUxNDk2ZmJhZTFiMjI4MDgyMDNiZDUiLCJwIjoiaiJ9)

#### Pull Request Creator Checklist

- [X] This PR has an associated issue or issues.
- [X] The associated issue(s) are linked above.
- [X] This PR meets all acceptance criteria for those issues.
- [X] This PR and linked issue(s) are adequately documented
- [X] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [X] Someone has been assigned this PR.
- [X] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

**Comment by abelzer212** [comment]
You probably shouldn't create your own version of the definition of population question. Rather I'd add some functionality in the existing question to address what you're adding, which seems to be a watch on the total which can be shown with your "watch" and with an optional flag that's passed in from the AMB measure to show it.

File: services/ui-src/src/measures/2021/AMBCH/questions/DefinitionsOfPopulation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+              displayValue:
+                "Managed Care Organization/Pre-paid Inpatient Health Plan (MCO/PIHP)",
+              value: DC.MCO_PIHP,
+              children: [
+                <CUI.Box pb="5" key="DeliverySys-MCO_PIHP-Percent">
+                  <QMR.NumberInput
+                    displayPercent
+                    mask={percentageAllowOneDecimalMax}
+                    formLabelProps={{ fontWeight: "400" }}
+                    label="What percent of your measure-eligible Managed Care Organization/Pre-paid Inpatient Health Plan (MCO/PIHP) population are included in the measure?"
```
Comment (abelzer212) [inline services/ui-src/src/measures/2021/AMBCH/questions/DefinitionsOfPopulation/index.tsx:277]: This text doesn't need to be different, it should match the other measures. James removed several things from this component per conversations with Marc and Karen, I would reach out to him to understand why these are gone and why you don't need to add them back

**Review by abelzer212 — CHANGES_REQUESTED** [review]

**Comment by abelzer212** [comment]
Screen Shot 2022-04-01 at 8 25 55 AM
Rate is times 1000 but I can only manually enter 2 digits for the rate field. I should be able to enter 0-1000, but I can only enter 0-99

**Comment by abelzer212** [comment]
Screen Shot 2022-04-01 at 8 28 40 AM
Screen Shot 2022-04-01 at 8 28 34 AM
I used the auto-calculations for all of the NDR Sets but I'm still getting the Total Rate error. 
This is probably a rounding issue, or we need to give this check some wiggle room. Probably + or - 1 but I'd talk with Marc and Karen on what they want

**Comment by jphechter** [comment]
I was unable to reproduce the validation error. Tried with the same inputs and it's all working now. It could be in part due to work Daniel had done on WCC-CH for the totaling portion of Rate.

**Comment by abelzer212** [comment]
Screen Shot 2022-04-05 at 8 57 22 AM
Screen Shot 2022-04-05 at 8 57 16 AM
It looks like now it's not triggering at all

**Comment by abelzer212** [comment]
The total Rate mismatch validation is also failing to trigger in OMS.

**Review by abelzer212 — APPROVED** [review]

**Pull request closed without merging**